rffmpeg.yml.sample

```diff
@@ -34,6 +34,9 @@ rffmpeg:
         # The remote SSH user to connect as
         user: jellyfin
 
+        # How long to persist SSH sessions (0 to disable)
+        persist: 120
```

Comment:
I'd be OK with the example default being even higher, maybe 300 seconds. But if you found this to be optimal, I'm good with it too!

Reply:
There really isn't a disadvantage to making it longer (as far as I'm aware of), I use 120 seconds on my setup mostly because that just addresses doing things like changing video quality or even switching movies shortly after starting playback. In theory, you could set it way longer, like an hour, and that way if you're binging a TV show, every ~20 minutes it uses the session, keeping the tunnel open longer. I'll make a commit to up this to 300, no reason not to.